packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js

```diff
@@ -84,6 +84,14 @@ function Unstable_TrapFocus(props) {
         return;
       }
 
+      // Support focused nodes removed from DOM with an interval call.
```

Comment:
I don't understand this comment. Why is it important whether the node was removed in an interval call or not?

Reply:
Yeah, it's not really removed the comment below is a bit misleading but became unfocusable is probably a better way of putting it.

Reply:
Thanks guys for the feedback

```suggestion
      // A focused element that become unfocusable doesn't fire any event we can listen to. Instead, the active element is moved to the body.
```

Reply:
Definitely better,